kevm-pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -141,8 +142,10 @@ def _get_method_abi(_mname: str) -> Dict:
         for msig in method_ids:
             mname = msig.split('(')[0]
             mid = int(method_ids[msig], 16)
-            _m = Contract.Method(mname, mid, _get_method_abi(mname), contract_name, self.sort_method)
-            _methods.append(_m)
+            _sort = _get_method_abi(mname)
```

Comment:
How about `sort` or `msort`?

Reply:
Instead of `_sort`?

Reply:
Yes.

Reply:
Updated to `msort` and tweaked the function to return `{}` instead of `None` since the function is supposed to return a `Dict`.